streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -1148,6 +1148,9 @@ private void verifyMaxInFlightRequestPerConnection(final Object maxInFlightReque
         consumerProps.put(REPLICATION_FACTOR_CONFIG, getInt(REPLICATION_FACTOR_CONFIG));
         consumerProps.put(APPLICATION_SERVER_CONFIG, getString(APPLICATION_SERVER_CONFIG));
         consumerProps.put(NUM_STANDBY_REPLICAS_CONFIG, getInt(NUM_STANDBY_REPLICAS_CONFIG));
+        consumerProps.put(ACCEPTABLE_RECOVERY_LAG_CONFIG, getLong(ACCEPTABLE_RECOVERY_LAG_CONFIG));
```

Comment:
Trying to avoid piling on even more unrelated questions to the thread below, but there's another config that would need to be passed in, the admin client timeout.
That said, can we just remove it? It only gets the timeout the admin is configured with anyway

Reply:
Ack, I'll take a look at it,.

Reply:
It does wind up "surviving" into the config that we use in the assignor, but I'm not sure if it's on purpose or just lucky.

I'll postpone any existential questions, and just add it to the regression test.

Reply:
Ah, it was also tested here: org.apache.kafka.streams.StreamsConfigTest#consumerConfigShouldContainAdminClientConfigsForRetriesAndRetryBackOffMsWithAdminPrefix

Reply:
I'll open a separate PR to remove the extra timeout config